### DISCUSSION TITLE
KYC 400 return (Duplicated)

### DISCUSSION THREAD

**Discussion by ivanwilliammd** [ivanwilliammd/satusehat-integration#115] [Q&A (Tanya Jawab)] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by uskaryulinanto** [comment]
saya sudah mengikuti langkah itu ... kalo di postman kode 200 tetapi response nya 400 mungkin karena payloadnya tidak cocok
[key gen.txt](https://github.com/user-attachments/files/18580976/key.gen.txt)
mohon bantuannya

**Reply by ivanwilliammd** [reply, marked as answer]
untuk KYC hanya bisa berjalan di environment production, silakan lihat thread ini
https://github.com/ivanwilliammd/satusehat-integration/discussions/132#discussioncomment-11914619
